server/package.json

```diff
@@ -44,17 +46,21 @@
     "unionized": "^4.12.0"
   },
   "scripts": {
-    "coverage": "./node_modules/nyc/bin/nyc.js --reporter=html --reporter=text --reporter=lcov --extension .es6 mocha",
-    "dev": "yarn; ./node_modules/.bin/sequelize db:migrate; ./node_modules/.bin/nodemon --harmony_default_parameters app.es6",
-    "lint": "eslint ./app.es6 ./models/noncommercial-application.es6 ./util.es6 ./test/noncommercial.js ./middlelayer-interaction.es6 ./test/test-util.js ./test/middlelayer-unit.js ./validation.es6 ./test/test-validate.js ./vcap-services.es6 ./temp-outfitter.es6 ./test/tempoutfitter-tests.js ./noncommercial.es6 ./application-general.es6",
+    "coverage":
+      "./node_modules/nyc/bin/nyc.js --check-coverage --lines 90 --reporter=html --reporter=text --reporter=lcov --extension .es6 mocha",
+    "dev":
+      "yarn; ./node_modules/.bin/sequelize db:migrate; ./node_modules/.bin/nodemon --harmony_default_parameters app.es6",
+    "lint":
+      "eslint ./app.es6 ./models/noncommercial-application.es6 ./util.es6 ./test/noncommercial.js ./middlelayer-interaction.es6 ./test/test-util.js ./test/middlelayer-unit.js ./validation.es6 ./test/test-validate.js ./vcap-services.es6 ./temp-outfitter.es6 ./test/tempoutfitter-tests.js ./noncommercial.es6 ./application-general.es6",
     "migrate": "./node_modules/.bin/sequelize db:migrate",
     "report-coverage": "CODECOV_TOKEN='e61f7ed1-686a-4f48-95b3-02fa7085995e' ./node_modules/nyc/bin/nyc.js report --reporter=lcov > coverage.lcov && codecov",
     "seed": "./node_modules/.bin/sequelize db:seed:all",
     "snyk-protect": "snyk protect",
-    "start": "node --harmony_default_parameters app.es6",
+    "start": "node --harmony_default_parameters ./src/app.es6",
```

Comment:
if you are now specifying `use-strict` at the top of all the files, do all the file types need to be `.es6`? Will that make it harder to run as `.es7`?